src/main/webapp/index.html

```diff
@@ -2,34 +2,121 @@
 <html>
   <head>
     <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="script.js"></script>
+    <title>Entertainment Hub - Dashboard</title>
   </head>
   <body onload="getDashboardItems()">
-    <label for="searchValue">Entertainment Item:</label>
-    <input
-      type="text"
-      name="searchValue"
-      id="searchValue"
-      oninput="getDashboardItems()"
-    />
-    <select
-      name="sortingDirection"
-      id="sortingDirection"
-      onchange="getDashboardItems()"
-      required
-    >
-      <option value="ASCENDING" selected
-        >Increasing Alphabetical Order (Title)</option
-      >
-      <option value="DESCENDING"
-        >Decreasing Alphabetical Order (Title)</option
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark mb-5">
+      <div class="container">
+        <a class="navbar-brand" href="#">Entertainment Hub</a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <input
+            class="form-control mr-2"
+            type="search"
+            name="searchValue"
+            id="searchValue"
+            placeholder="Search by Title"
+            oninput="getDashboardItems()"
+          />
+          <select
+            class="form-control mr-2 w-auto"
+            name="sortingDirection"
+            id="sortingDirection"
+            onchange="getDashboardItems()"
+            required
+          >
+            <option value="ASCENDING" selected
+              >Increasing Alphabetical Order (Title)</option
+            >
+            <option value="DESCENDING"
+              >Decreasing Alphabetical Order (Title)</option
+            >
+          </select>
+          <button
+            type="button"
+            class="btn btn-light navbar-btn text-nowrap"
+            data-toggle="modal"
+            data-target="#itemSubmissionDialog"
+          >
+            Add New Entertainment Item
+          </button>
+        </div>
+      </div>
+    </nav>
+    <div class="container">
+      <div
```

Comment:
you can move the dialog html code into its own .html file and then load it here so this one doesn't get bloated.

Reply:
I moved dialog code into a separate file and loaded it into index.html in last commit.